EmployeeLeaveTracking/EmailService/EmailHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Net.Mail;
+using System.Net;
+
+namespace EmailService
+{
+    public class EmailHelper
+    {
+
+        public void SendEmail(string emailToAddress,string subject, string body)
+        {
+            using MailMessage mail = new();
+            string smtpAddress = "smtp.gmail.com";
+            int portNumber = 587;
+            bool enableSSL = true;
+            string emailFromAddress = "preetyraj340@gmail.com"; //sender email address  
+            string password = "bjnuvxfkcbwqmmye"; //sender password  
+
+            mail.From = new MailAddress(emailFromAddress);
+            mail.To.Add(emailToAddress);
+            mail.Subject = subject;
+            mail.Body = body;                           
+            mail.IsBodyHtml = true;
```

Comment:
What is meaning on IsBodyHtml? What is responsibility of this variable

Reply:
IsBodyHtml property indicates that the body of the email is either HTML content or plain text.
Using HTML content we can include Customized and Dynamic layouts with text, images, hyperlinks, and other elements in the email's body.